frontend/src/summary.pug

```diff
@@ -90,30 +90,25 @@
       .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}
   #summary-charts
     .summary-charts(v-for="(repo, i) in filteredRepos")
-      .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'")
+      .summary-charts__title(v-if="filterGroupSelection !== 'groupByNone'")
         .summary-charts__title--index {{ i+1 }}
-        .summary-charts__title--repo {{ repo[0].repoName }}
+        .summary-charts__title--repo
```

Comment:
Just want to clarify, is there a reason we do not name this `.summary-charts__title--group` as it can be used for both grouping by authors or repos?

Reply:
I don't think so. I agree `...--group` is a better name. Will change accordingly.